src/dialog/reimu.c

```diff
@@ -916,8 +918,7 @@ DIALOG_TASK(reimu, StageExPostBoss) {
 
 	FACE(reimu, assertive);
 	MSG(reimu, "Do something useful with that big brain of yours for once!");
-	FACE(reimu, irritated);
-	MSG(reimu, "Count yourself lucky that Kanako probably won’t put you on fairy vomit duty!");
+	MSG(reimu, "Count yourself lucky that Kanako probably won’t have you cleaning fairy vomit like your former underlings!");
```

Comment:
I found the original line pretty funny, not sure it needs any more elaboration to work.